cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -637,7 +637,7 @@ public String getFormat() {
             EXPORT_ALL_LIMITS_GROUP,
             ParameterType.BOOLEAN,
             "True to export all OperationalLimitsGroup, False to export only the selected group",
-            CgmesExportContext.EXPORT_LOAD_FLOW_STATUS_DEFAULT_VALUE);
+            CgmesExportContext.EXPORT_ALL_LIMITS_GROUP_DEFAULT_VALUE);
```

Comment:
Both `EXPORT_LOAD_FLOW_STATUS_DEFAULT_VALUE` and `EXPORT_ALL_LIMITS_GROUP_DEFAULT_VALUE` have the same value (`true`). It is just a quality change since the latter should have been used here in the 1st place (default value for `EXPORT_ALL_LIMITS_GROUP` parameter).
=> OK